libexec/scoop-import.ps1

```diff
@@ -59,6 +63,14 @@ foreach ($item in $import.apps) {
 
     & "$PSScriptRoot\scoop-install.ps1" $app @instArgs
 
+    if ($update) {
+        & "$PSScriptRoot\scoop-update.ps1" $app$global
+    }
+
+    if ($reset) {
+        & "$PSScriptRoot\scoop-reset.ps1" $app
+    }
```

Comment:
Maybe it should not, but it is. As I described in the issue https://github.com/ScoopInstaller/Scoop/issues/5525 if install different versions of the same app, no install or update command gives you what you want.

Example: I have the apps python310 and python311 out of the versions bucket installed. python311 was the last one and so the shim python.exe points to that of python311. If I now import a scoopfile.json refering to python310, nothing is changed during import. I end up with the wrong python.exe in the PATH. Therefore the reset argument.

Reply:
This happens after both `python310` and `python311` have been installed already, then the import operation is performed right?

Reply:
Exactly.

Reply:
@xxthunder technically, this is also not required. To see what I mean:  

https://github.com/ScoopInstaller/Scoop/blob/6cdcc75ad84add9deedad81f1c9c16dc43f868c0/lib/install.ps1#L58-L66 

The functions that `scoop-reset` will run (see below) have already been run by `scoop-install` (see above). 

https://github.com/ScoopInstaller/Scoop/blob/6cdcc75ad84add9deedad81f1c9c16dc43f868c0/libexec/scoop-reset.ps1#L80-L87

Reply:
@rashil2000 @HUMORCE I think this is not true. I want the reset no matter what is already installed. So if I import with, e.g., cmake version 3.24 in my scoopfile.json and cmake 3.24 and 3.26 both already installed and p3.26 was installed the last time. Then the shim cmake points to cmake 3.26. Now I can import my scoopfile as often as I want, nothing will be installed because 3.24 is already installed and nothing will be resetted. So the shim cmake points to cmake 3.26. Exactly this my problem with scoop import. My option --reset solves this.